### PR TITLE
En 1748 improvement of page's group handling (version 4.3.4)

### PR DESCRIPTION
**Pull request by eugeniosant**

### CODE REVIEW

**Comment by entando-service** [comment]
All is well

**Comment by entando-service** [comment]
All is well

**Comment by entando-service** [comment]
All is well

File: admin-console/src/main/java/com/agiletec/apsadmin/portal/PageAction.java [hunk 1]
```diff
@@ -159,15 +171,14 @@ public String settingsPage() {
...
         boolean groupSelectLock = false;
         this.setStrutsAction(ApsAdminSystemConstants.ADD);
         if (parentPage != null) {
             this.setParentPageCode(parentPage.getCode());
-            groupName = parentPage.getGroup();
+            String groupName = parentPage.getGroup();
             this.setGroup(groupName);
             boolean isParentFree = Group.FREE_GROUP_NAME.equals(groupName);
-            groupSelectLock = !(this.isCurrentUserMemberOf(Group.ADMINS_GROUP_NAME) && isParentFree);
+            groupSelectLock = !isParentFree && !this.isCurrentUserMemberOf(Group.ADMINS_GROUP_NAME);
```
Comment (joewhite101) [inline admin-console/src/main/java/com/agiletec/apsadmin/portal/PageAction.java]: This is logically different. Just want to make sure that is intentional to fix something?

**Pull request merged**